StreamChatUITestsAppUITests/MockServer/MessageResponses.swift

```diff
@@ -10,103 +10,146 @@ extension StreamMockServer {
     
     func configureMessagingEndpoints() {
         server[MockEndpoint.message] = { [weak self] request in
-            self?.messageCreation(request: request) ?? .badRequest(nil)
+            let channelId = try! XCTUnwrap(request.params[EndpointQuery.channelId])
+            return self?.messageCreation(request, channelId: channelId) ?? .badRequest(nil)
         }
         server[MockEndpoint.messageUpdate] = { [weak self] request in
-            self?.messageUpdate(request: request) ?? .badRequest(nil)
+            self?.messageUpdate(request) ?? .badRequest(nil)
         }
         server[MockEndpoint.replies] = { [weak self] request in
-            self?.mockMessageReplies(request: request) ?? .badRequest(nil)
+            let messageId = try! XCTUnwrap(request.params[EndpointQuery.messageId])
+            return self?.mockMessageReplies(messageId) ?? .badRequest(nil)
         }
     }
     
-    func mockDeletedMessage(_ message: [String: Any]) -> [String: Any] {
+    func mockDeletedMessage(_ message: [String: Any], user: [String: Any]) -> [String: Any] {
         var mockedMessage = message
         mockedMessage[MessagePayloadsCodingKeys.deletedAt.rawValue] = TestData.currentDate
         mockedMessage[MessagePayloadsCodingKeys.type.rawValue] = MessageType.deleted.rawValue
+        mockedMessage[MessagePayloadsCodingKeys.user.rawValue] = user
         return mockedMessage
     }
     
     func mockMessage(
         _ message: [String: Any],
+        channelId: String,
         messageId: String?,
         text: String?,
+        user: [String: Any],
         createdAt: String?,
-        updatedAt: String?,
-        intercept: ((inout [String: Any]) -> [String: Any])? = nil
+        updatedAt: String?
     ) -> [String: Any] {
         var mockedMessage = message
+        let channelType = ChannelType.messaging.rawValue
+        mockedMessage[MessagePayloadsCodingKeys.cid.rawValue] = "\(channelType):\(channelId)"
+        mockedMessage[EventPayload.CodingKeys.channelId.rawValue] = channelId
         mockedMessage[MessagePayloadsCodingKeys.id.rawValue] = messageId
         mockedMessage[MessagePayloadsCodingKeys.createdAt.rawValue] = createdAt
         mockedMessage[MessagePayloadsCodingKeys.updatedAt.rawValue] = updatedAt
         mockedMessage[MessagePayloadsCodingKeys.text.rawValue] = text
         mockedMessage[MessagePayloadsCodingKeys.html.rawValue] = text?.html
-        mockedMessage = intercept?(&mockedMessage) ?? mockedMessage
+        mockedMessage[MessagePayloadsCodingKeys.user.rawValue] = user
```

Comment:
Let's rename the `MessagePayloadsCodingKeys.user` to `MessagePayloadsCodingKeys.users`

Reply:
@bielikb, same here, I think `user` is grand. Btw, `MessagePayloadsCodingKeys` is a source code. Thoughts?

Reply:
@alteral does it corresponds to exactly 1 `user` or is this an array?

Reply:
This is an object in the backend response. It corresponds to exactly 1 `user`. E.g.:
```json
"user": {
      "id": "luke_skywalker",
      "role": "user",
      "created_at": "2020-12-07T11:36:47.059906Z",
      "updated_at": "2022-03-08T15:30:07.960068Z",
      "last_active": "2022-03-08T15:57:33.831192082Z",
      "banned": false,
      "online": true,
      "image": "https://blahblahblah.com/1.jpg",
      "birthland": "Tatooine",
      "name": "Luke Skywalker"
    }
```